quelpa.el

```diff
@@ -1784,6 +1784,23 @@ the `quelpa' command has been run in the current Emacs session."
                 (quelpa item)))
             quelpa-cache))))
 
+;;;###autoload
+(defun quelpa-upgrade (rcp)
+  "Upgrade a package found in `quelpa-cache' with recipe RCP."
+  (interactive
+   (when (quelpa-setup-p)
+     (let* ((quelpa-melpa-recipe-stores (list quelpa-cache))
+            (name (quelpa-interactive-candidate))
+            (prefix (prefix-numeric-value current-prefix-arg)))
+       (list (assoc name quelpa-cache)))))
+  (when rcp
+    (let* ((quelpa-upgrade-p t)
+           (current-prefix-arg nil))
```

Comment:
As we discussed in the other PR, all the interactive stuff should be handled in the `interactive` form, so `current-prefix-arg` shouldn't be touched here.

Reply:
`(quelpa rcp)` is actively checking `current-prefix-arg`, do you want to change behavior of that function too?

Reply:
I think you mean this binding in `quelpa`:

```el
(quelpa-upgrade-p (if current-prefix-arg t quelpa-upgrade-p))
```
If `current-prefix-arg` is nil, it falls back to `quelpa-upgrade-p`, which is still being bound.